drivers/overlay/ov_network.go

```diff
@@ -183,9 +171,36 @@ func (n *network) initSandbox() error {
 
 	go n.watchMiss(nlSock)
 
+	if n.driver.serfInstance != nil {
+		return n.initVxlan()
```

Comment:
Why should we delay vxlan creation if serf is not initialized? We should only delay peerDb update